core/constants.go

```diff
@@ -327,3 +327,9 @@ const NodesCoordinatorRegistryKeyPrefix = "indexHashed_"
 
 // HashSizeInBytes represents the size in bytes of a hash
 const HashSizeInBytes = 32
+
+// SignatureSizeInBytes represents the size in bytes of a BLS signature
+const SignatureSizeInBytes = 48
```

Comment:
These consts should not be here. It will be impossible to change the public key type and the signature type from config

Reply:
I would pull them in config.toml in [Consensus] section